src/Elm/Aeson.hs

```diff
@@ -73,13 +71,13 @@ it is decoded it the following way for each of the specified types:
 
 >>> data User = User { userName :: String, userAge :: Int } deriving (Generic, Show)
 >>> instance FromJSON User where parseJSON = elmStreetParseJson
->>> decode @User "{ \"name\": \"John\", \"age\": 42 }"
+>>> decode @User "{\"age\":42,\"name\":\"John\",\"tag\":\"User\"}"
 Just (User {userName = "John", userAge = 42})
 
 
 >>> data VeryLongType = VeryLongType { vltName :: String, vltAge :: Int } deriving (Generic, Show)
 >>> instance FromJSON VeryLongType where parseJSON = elmStreetParseJson
->>> decode @VeryLongType "{ \"name\": \"John\", \"age\": 42 }"
+>>> decode @VeryLongType "{\"age\":42,\"name\":\"John\",\"tag\":\"VeryLongType\"}"
```

Comment:
doesn't seem that long to me :eyes: `ProjectContractChargingPeriodProjectAccountReferenceVMFactoryBuilderStrategy`